develop/events.rst

```diff
@@ -112,3 +112,72 @@ Regular expression triggered events
 
 The :class:`supybot.callbacks.PluginRegexp` class provides some utilities
 for creating plugins that act on regular expression matching.
+
+Reacting to URLs
+----------------
+
+Using Limnoria's ``urlSnarfer`` will avoid loops with other bots and ignore private messages.
+
+.. code-block:: python
+
+    # An example plugin using urlSnarfer
+    # https://docs.limnoria.net/develop/commands.html#supybot.commands.urlSnarfer
```

Comment:
Rather than embedding links in the code, it's probably better to add descriptions before or after the code sample. That way, you can use inline links to the references.